src/Repository/ActivityRepository.php

```diff
@@ -149,4 +155,23 @@ public function getPaginatedActivities(
 
         return $query;
     }
+
+    public function getActivitiesForValidation(
+        User $user
+    ): QueryBuilder {
+        $queryBuilder = $this->createQueryBuilder('activity');
+        $queryBuilder
+            ->select('activity')
+            ->join('activity.owner', 'user')
+            ->where('activity.status = :in_validation')
+            ->setParameter(':in_validation', Activity::STATUS_IN_VALIDATION);
+        $hasAccess = $this->checker->isGranted('ROLE_ADMIN');
```

Comment:
Everywhere the variable name is `$hasAccess`! Why not use something more explicit like `$isAdmin`.